### ISSUE TITLE
PortableInfobox: <title> tag renders <p> outside of <h2>

### ISSUE THREAD

**Issue by DeadSuperHero** [StarCitizenTools/mediawiki-skins-Citizen#553]
Hey, thanks for your hard work on such an amazing theme. I'm having this really weird issue with PortableInfobox and this theme. I don't know if this is just my own bad markup, or what, but I have this really weird issue with my Title element when using PortableInfobox with this theme.

Before I begin, my MediaWiki version is 

Here's my markup for the infobox:
```
<infobox>
  <title source="title">
    <default>{{PAGENAME}}</default>
  </title>
    <image source="image">
        <caption source="imagecaption" />
    </image>
    <data source="found">
        <label>Founded</label>
    </data>
    <data source="ind">
        <label>Industry</label>
    </data>
    <data source="hq">
        <label>Location</label>
    </data>
</infobox>
```
Pretty straightforward, nothing too complicated. Same goes with the wikitext for a page entry:

```
{{infobox brand
 | title         = Headless Corporation
 | image         = headless_logo.png
 | imagecaption  = Current modern corporate branding for Headless
 | found           = 1985
 | ind           = Artificial Intelligence
 | hq      = Wilmington, Delaware
}}
```

Overall, when I save my page for the first time, it looks pretty great:
![Screenshot_20221122_133409](https://user-images.githubusercontent.com/401560/203415858-6d890ec8-28e1-474f-99f0-0fa5dadd807b.png)
Note the HTML structure as it gets saved:
```
<h2 class="pi-item pi-item-spacing pi-title" data-source="title">
  <p>Headless Corporation</p>
</h2>
```

However, when I come back to the page, even after a refresh, this happens:
![Screenshot_20221122_133635](https://user-images.githubusercontent.com/401560/203416141-02a5bcf4-b7b6-48c1-b338-ab72dc81504e.png)

Here's what the HTML structure looks like now:
```
<h2 class="pi-item pi-item-spacing pi-title" data-source="title"></h2>
<p>Headless Corporation</p>
```

So basically, somehow the paragraph element gets shifted around in the DOM so that `<p>` is no longer inside `<h2>`. My guess is that something gets preprocessed with the Visual Editor when I save any kind of change to the Infobox, but doesn't get applied correctly after saving. After disabling VE and trying again, the problem persists.

**Comment by alistair3149** [comment]
Hmm this is probably an upstream issue with PortableInfobox since Citizen does not change the DOM element of PortableInfobox. You might want to submit a ticket [here](https://github.com/Universal-Omega/PortableInfobox/issues).

**Comment by alistair3149** [comment]
I stumbled upon this solution on the issue tracker of Portable Infobox:
https://github.com/Universal-Omega/PortableInfobox/issues/81

May be this will fix the problem for you?

**Comment by alistair3149** [comment]
Closing since it is not related to Citizen, but PortableInfobox and the Remex parser: https://github.com/Universal-Omega/PortableInfobox/issues/81
